### DISCUSSION TITLE
記事内のリンクが切れていました

### DISCUSSION THREAD

**Discussion by ericgpks** [rubima/magazine.rubyist.net#521] [記事についての報告]
https://magazine.rubyist.net/articles/prerubykaigi2011/preRubyKaigi2011-index.html
と
https://magazine.rubyist.net/articles/prerubykaigi2011/preRubyKaigi2011-04
にて
RubyKaigi2011直前特集号の記事へのリンクが壊れていたので修正しました。
以下にPRとして作成しております。
https://github.com/rubima/magazine.rubyist.net/pull/520
ご確認をお願いします。

**Comment by neko314** [comment]
こちらありがとうございました 🙏
